tests/test_h3.py

```diff
@@ -197,190 +231,301 @@ def test_polyfill_bogus_geo_json(self):
+                            -64.742431640625, -55.41654360858007
+                        ], [-67.642822265625, -55.41654360858007]
+                    ]
+                ]
+            }, 9, True
+        )
         self.assertGreater(
-            len(hexagons), 10, 'got an appropriate number of hexagons back')
+            len(hexagons), 10, 'got an appropriate number of hexagons back'
+        )
 
     def test_polyfill_null_island(self):
-        hexagons = h3.polyfill({
-            "type":
-            "Polygon",
-            "coordinates": [[[-3.218994140625, -3.0856655287215378], [
-                -3.218994140625, 3.6888551431470478
-            ], [3.5815429687499996, 3.6888551431470478], [
-                3.5815429687499996, -3.0856655287215378
-            ], [-3.218994140625, -3.0856655287215378]]]
-        }, 4, True)
+        hexagons = h3.polyfill(
+            {
+                "type":
+                "Polygon",
+                "coordinates": [
+                    [
+                        [-3.218994140625, -3.0856655287215378], [
+                            -3.218994140625, 3.6888551431470478
+                        ], [3.5815429687499996, 3.6888551431470478], [
+                            3.5815429687499996, -3.0856655287215378
+                        ], [-3.218994140625, -3.0856655287215378]
+                    ]
+                ]
+            }, 4, True
+        )
         self.assertGreater(
-            len(hexagons), 10, 'got an appropriate number of hexagons back')
+            len(hexagons), 10, 'got an appropriate number of hexagons back'
+        )
 
     def test_h3_set_to_multi_polygon_empty(self):
         h3_addresses = []
         multi_polygon = h3.h3_set_to_multi_polygon(h3_addresses)
 
-        self.assertEqual(multi_polygon, [],
-                         'no hexagons yields an empty array')
+        self.assertEqual(multi_polygon, [], 'no hexagons yields an empty array')
 
     def test_h3_set_to_multi_polygon_single(self):
         h3_addresses = ['89283082837ffff']
         multi_polygon = h3.h3_set_to_multi_polygon(h3_addresses)
         vertices = h3.h3_to_geo_boundary(h3_addresses[0])
-        # This is tricky, because output in an order starting from any vertex
-        # would also be correct, but that's difficult to assert and there's
-        # value in being specific here
-        expected = [[[
-            vertices[2], vertices[3], vertices[4], vertices[5], vertices[0],
-            vertices[1]
-        ]]]
+        expected = [
+            [
+                [
+                    vertices[2], vertices[3], vertices[4], vertices[5],
+                    vertices[0], vertices[1]
+                ]
+            ]
+        ]
+
+        # We shift the multi_polygon circular list so that it starts from expected[0][0][0],
+        # since output starting from any vertex would be correct as long as it's in order.
+        multi_polygon[0][0] = self.shift_circular_list(
```

Comment:
It seems like it would be a lot easier just to manually change the order in `expected`? In general I try to avoid extra data transformations in tests, as it makes them harder to debug.

It might actually feel more valid here to try every starting node, and assert that at least one was correct (closer to property-based testing). Either way, this is up to you - I think it's okay like this if you prefer.

Reply:
That makes sense. The problem I found is once the core algorithm changes, the order of polygon coords may change which makes h3_set_to_multi_polygon tests very brittle.

Yes, I will find a way to configure the code formatter since extraneous formatting change like this is really distracting.